### ISSUE TITLE
Error while compiling on Ubuntu 18.04 LTS

### ISSUE THREAD

**Issue by tibixdj** [ElvishArtisan/rivendell#547]
after `./configure`
I get this error:
```
configure:16774: checking for MUSICBRAINZ
configure:16781: $PKG_CONFIG --exists --print-errors "libdiscid libmusicbrainz5 libcoverart"
Package libdiscid was not found in the pkg-config search path.
Perhaps you should add the directory containing `libdiscid.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libdiscid' found
Package libcoverart was not found in the pkg-config search path.
Perhaps you should add the directory containing `libcoverart.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libcoverart' found
configure:16784: $? = 1
configure:16815: result: no
No package 'libdiscid' found
No package 'libcoverart' found
configure:16831: error: *** LibMusicBrainz not found ***

```
and far down in same config.log
```
MUSICBRAINZ_CFLAGS=''
MUSICBRAINZ_LIBS='/usr/local/musicbrainz5:'
```

**Comment by suehring-uwsp** [comment]
Ran into this as well.  The libmusicbrainz check within the configure script is actually checking for three different packages that do not appear to have cross-dependencies.  Therefore, the solution for me was to install libdiscid and libcoverart.  I believe I needed to install the -devel version of each, as in "libdiscid-devel" and "libcoverart-devel" but I don't have the VM fired up at the moment to verify if it was the -devel version or not for both of those packages.

Steve
